matcalc/relaxation.py

```diff
@@ -7,7 +7,7 @@
 import pickle
 from typing import TYPE_CHECKING
 
-from ase.constraints import ExpCellFilter
+from ase.filters import ExpCellFilter
```

Comment:
let's change this to the new released `FrechetCellFilter` while we're at it which has [improved convergence behavior](https://gitlab.com/ase/ase/-/merge_requests/3024)

Reply:
also, could you recreate the figures since the relaxation times might decrease a bit?

Reply:
Can do

Reply:
@janosh Happy to implement this. Implementing this however, will require me to rewrite some of the tests as some of the results will be different with the `FrechetCellFilter`.

For `PropCalc` classes which have the option `relax_structure`, I think an option to pass kwargs to the `RelaxCalc`, if the user desires. Happy to also implement this too!

Reply:
both would be great! 🙏

Reply:
Added the relax kwargs to the other `PropCalc` classes, and have tested this using the tests by passing `{"cell_filter", ExpCellFilter}` to get the same results prior to making `FrechetCellFilter` the default. I've also added some tests in `test_relaxation.py` to test the outputs of the new default `FrechetCellFilter`.